gathering/uk/about/the-team.md

```diff
@@ -3,3 +3,37 @@ layout: gathering
 title: The team
 ---
 
+MathsJam Gathering is a fully volunteer-run event, and many people play an important role in making the event a success. The people listed on this page form the core orgnisational team and lead the organisation of the event.
+
+### Colin Wright
+<img src="Colin.jpg" class="team-pic"></img>
+
+Colin has a Ph.D. from Cambridge and has worked as a research mathematician, and done research in computing. His day job is developing systems to do the maritime version of Air Traffic Control, and in his "free time" he travels around the world giving lectures on how and why maths can be interesting, fun, useful and even on occasion, exciting.
+
+### Sam Hartburn
```

Comment:
Need to fill in this section for everyone, and add whoever I've missed including